packages/bot-web-ui/src/stores/download-store.js

```diff
@@ -0,0 +1,109 @@
+import { action } from 'mobx';
+import { localize } from '@deriv/translations';
+
+export default class DownloadStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    getSuccessJournalMessage = (message, extra) => {
+        const { profit, sold_for, longcode, transaction_id } = extra;
+        switch (message) {
+            case 'BLOCK_LOADED': {
```

Comment:
- We should compare with values found in `log_types` in file `deriv-app/packages/bot-skeleton/src/constants/messages.js` rather than hardcoding these values.
- If we're comparing with `log_types` there's no need for us to call `toUpperCase` on line 81.

Reply:
Changed required `log_types` in messages.js and this has been implemented. :)